hack/quickcluster/README.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Configuring NFS storage provisioner on Quickcluster clusters
+---
+
+This document and script are based on the following documents:
+
+- https://two-oes.medium.com/openshift-4-8-and-above-with-nfs-subdir-external-provisioner-9b6b614194b7
+- https://source.redhat.com/groups/public/api-management-sbr/cee_sd_api_management_wiki/configuring_nfs_for_persistent_volumes
+
+This procedure will essentially deploy a new storage provisioner to your cluster.
+
+###### Requirements 
+
+This script can only be executed once:
+ 
+- your cluster has been provisioned.
+- openshift has been installed.
+
+###### Input
+
+This script takes 1 parameter - the UPI hostname in a Resource Hub Quickcluster.
+It can be found in the Cluster Information section.
```

Comment:
```suggestion
This script takes 1 mandatory  parameter - the UPI hostname in a Resource Hub Quickcluster.
It can be found in the Cluster Information section.
It takes another optional parameter - a path to an SSH private key to be used for authenticating with
the host. When not provided, the script will use instead the private key at `~/.ssh/id_rsa`.
```

Reply:
Nice catch. Fixed